platform/android/app/src/main/kotlin/coop/polypoly/polypod/network/Network.kt

```diff
@@ -19,58 +19,7 @@ open class Network(val context: Context) {
 
     data class NetworkResponse(var data: String?, var error: String?)
 
-    open suspend fun httpPost(
-        url: String,
-        body: String,
-        contentType: String?,
-        authToken: String?,
-        allowInsecure: Boolean
-    ): NetworkResponse = withContext(Dispatchers.IO) {
-        var response = NetworkResponse(data = null, error = null)
-        val connection = httpConnection(
-            "POST", url, body, contentType, authToken, allowInsecure
-        ) ?: return@withContext NetworkResponse(
-            null, "network connection failed"
-        )
-
-        try {
-            val responseCode = connection.responseCode
-            response.error = validateResponseCode(responseCode)
-            response.data =
-                connection.inputStream.bufferedReader().use { it.readText() }
-        } finally {
-            connection.disconnect()
-            return@withContext response
-        }
-        return@withContext response
-    }
-
-    open suspend fun httpGet(
-        url: String,
-        contentType: String?,
-        authToken: String?,
-        allowInsecure: Boolean
-    ): NetworkResponse = withContext(Dispatchers.IO) {
-        var response = NetworkResponse(data = null, error = null)
-        val connection = httpConnection(
-            "GET", url, null, contentType, authToken, allowInsecure
-        ) ?: return@withContext NetworkResponse(
-            null, "network connection failed"
-        )
-
-        try {
-            val responseCode = connection.responseCode
-            response.error = validateResponseCode(responseCode)
-            response.data =
-                connection.inputStream.bufferedReader().use { it.readText() }
-        } finally {
-            connection.disconnect()
-            return@withContext response
-        }
-        return@withContext response
-    }
-
-    fun validateResponseCode(responseCode: Int): String? {
+    private fun validateResponseCode(responseCode: Int): String? {
```

Comment:
the diff is not helpful here.
only change: set `validateResponseCode` and `httpConnection` as private and move them to the top.